confgenerator/testdata/valid/windows_default_config/golden_fluent_bit_main.conf

```diff
@@ -0,0 +1,160 @@
+    # Enable a built-in HTTP server that can be used to query internal information and monitor metrics of each running plugin.
+    HTTP_Server  On
+    HTTP_Listen  0.0.0.0
+    HTTP_PORT    2020
+
+    # https://docs.fluentbit.io/manual/administration/buffering-and-storage#service-section-configuration
+    # storage.path is set by Fluent Bit systemd unit (e.g. /var/lib/google-cloud-ops-agent/fluent-bit/buffers).
+    storage.sync               normal
+    # Enable the data integrity check when writing and reading data from the filesystem.
+    storage.checksum           on
+    # The maximum amount of data to load into the memory when processing old chunks from the backlog that is from previous Fluent Bit processes (e.g. Fluent Bit may have crashed or restarted).
+    storage.backlog.mem_limit  50M
+    # Enable storage metrics in the built-in HTTP server.
+    storage.metrics            on
+    # This is exclusive to filesystem storage type. It specifies the number of chunks (every chunk is a file) that can be up in memory.
+    # Every chunk is a file, so having it up in memory means having an open file descriptor. In case there are thousands of chunks,
+    # we don't want them to all be loaded into the memory.
+    storage.max_chunks_up      128
+
+[INPUT]
+    # https://docs.fluentbit.io/manual/pipeline/inputs/tail#config
+    Name               tail
+    DB                 /var/lib/google-cloud-ops-agent/fluent-bit/buffers/ops-agent-fluent-bit
+    Path               /var/log/google-cloud-ops-agent/subagents/logging-module.log
+    Tag                ops-agent-fluent-bit
+    # Set the chunk limit conservatively to avoid exceeding the recommended chunk size of 5MB per write request.
+    Buffer_Chunk_Size  512k
+    # Set the max size a bit larger to accommodate for long log lines.
+    Buffer_Max_Size    5M
+    # When a message is unstructured (no parser applied), append it under a key named "message".
+    Key                message
+    # Increase this to 30 seconds so log rotations are handled more gracefully.
+    Rotate_Wait        30
+    # Skip long lines instead of skipping the entire file when a long line exceeds buffer size.
+    Skip_Long_Lines    On
+
+    # https://docs.fluentbit.io/manual/administration/buffering-and-storage#input-section-configuration
+    # Buffer in disk to improve reliability.
+    storage.type       filesystem
+
+    # https://docs.fluentbit.io/manual/administration/backpressure#mem_buf_limit
+    # This controls how much data the input plugin can hold in memory once the data is ingested into the core.
+    # This is used to deal with backpressure scenarios (e.g: cannot flush data for some reason).
+    # When the input plugin hits "mem_buf_limit", because we have enabled filesystem storage type, mem_buf_limit acts
+    # as a hint to set "how much data can be up in memory", once the limit is reached it continues writing to disk.
+    Mem_Buf_Limit      10M
+
+[INPUT]
```

Comment:
this blob for collectd is no-op. I have a follow-up PR to clean it up once we introduce a isWindows flag to handle the   difference between Linux and Windows (also Path name)

Reply:
This will be a follow up PR adding isWindows flag